tests/dialects/test_duckdb.py

```diff
@@ -359,6 +359,16 @@ def test_duckdb(self):
 
         self.validate_identity("SELECT ISNAN(x)")
 
+        self.validate_all(
+            "MEDIAN(amount)",
+            write={
+                "postgres": "PERCENTILE_CONT(0.5) WITHIN GROUP (ORDER BY amount)",
+                "presto": "APPROX_PERCENTILE(amount, 0.5)",
+                "spark": "PERCENTILE_APPROX(amount, 0.5)",
+                "duckdb": "APPROX_QUANTILE(amount, 0.5)",
```

Comment:
DuckDB's [docs](https://duckdb.org/docs/sql/aggregates) mention that the formula for `MEDIAN(x)` is `QUANTILE_CONT(x, 0.5)`; is the transformation to `APPROX_QUANTILE(x, 0.5)` lossless / equivalent to the `CONT` variant? Are `NULL` values handled differently?